tests/test_common.c

```diff
@@ -124,3 +124,9 @@ test_async_ctxt_get(trunk_handle      *spl,
 
    return ctxt;
 }
+
+inline bool
+show_verbose_test_progress(test_exec_config *test_exec_cfg)
```

Comment:
Maybe this should move to `.h` as it's defined as `inline` ? Wasn't too sure ... ???

Reply:
This should be `test_show_verbose_progress`, since it's exposed and coming from a test header.

Reply:
I had it named like that before, but because of this nagging `test_` and then `show_` terminology, I felt the current name read better.

Anyway ... I'll change it as you remarked, but it's not my 1st choice.